source/developer/mobile-developer-setup.rst

```diff
@@ -252,6 +253,20 @@ These make commands are used to ensure that the code follows the linter rules an
 
  - **make check-style**: Runs the ESLint JavaScript linter.
  - **make test**: Runs the tests.
+ 
+ Commands to Build the App
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The set of commands for building the app are used in conjunction with `*Fastlane* <https://fastlane.tools/>`_. For most of them, you will need to modify `Fastfile <link>`_, since the commands are coupled with Mattermost's build and deployment process.
+
+You will always be able to build an unsigned version of the app as it does not need provisioning profiles or certificates as long as you set up *Fastlane* in your environment.
+
+ - **make build-ios**: Builds the iOS app and generates the *.ipa* file to be distributed. This make command expects an argument as the target which can be *dev*, *beta* or *release*. Depending on the target, a Fastlane script runs and each lane has the appropriate certificates and steps according to the Mattermost release process.
+ - **make build-android**: Builds the Android app and generates the *.apk* file to be distributed. This make command expects an argument as the target which can be *dev*, *alpha* or *release*. Depending on the target, a Fastlane script runs and each lane has the appropriate certificates and steps according to the Mattermost release process.
+ - **make unsigned-ios**: Builds the iOS app and generates an unsigned *Mattermost-unsigned.ipa* file in the project's root directory.
+ - **make unsigned-android**: Builds the Android app and generates an unsigned *Mattermost-unsigned.apk* file in the project's root directory.
+
+If you plan to use the **make build-*** commands, be sure to modify Fastlane to suit your needs or the commands will fail.
```

Comment:
"be sure to modify Fastlane to suit your needs or the commands will fail". --> is there an example we can give? @enahum

Reply:
they can look how to do it here https://docs.fastlane.tools/